autoload/neomake/signs.vim

```diff
@@ -125,8 +125,15 @@ function! neomake#signs#PlaceVisibleSigns() abort
         if !has_key(s:sign_queue[type], buf)
             continue
         endif
-        let topline = line('w0')
-        let botline = line('w$')
+
+        if !get(g:, 'neomake_place_signs_at_once', 0)
+            let topline = line('w0')
+            let botline = line('w$')
+        else
+            let topline = line('0')
+            let botline = line('$')
```

Comment:
I agree.
@ivalkeen what do you think?

Does an option for this then still makes sense (to get the current behavior)?
I could imagine that it's better performance-wise with a lot of signs, but on the other hand, the lazy placing gets called on every cursor move..